content/en/event-streaming.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Event Streaming
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+Event streaming is a system in which one is providing a constant flow of data.
+The data in this stream contains information regarding events (changes of state)
+
+## Problem it addresses
+
+As more data becomes real-time, there will be an increased need to ensure that data is consistently flowing from a producer (source) to a consumer (receiver).
+Past architectures would pull data in small batches over set period of time but nowadays, applications are needing to get constant, up-to-date data.
+
+## How it helps
+
+Event streaming can be seen as an extension of [Event-driven architectures](/event-driven-architecture/) while event-driven architecture is the infrastructure designed to support the streaming of events to their respective services.
```

Comment:
Isn't event streaming how event-driven architectures are implemented? If so, I wouldn't call it an extension. 

```suggestion
Event streaming is how services in an [event-driven architectures](/event-driven-architecture/) communicate. Hence, the former is the process, and the latter is the infrastructure to support that process. 
```